### DISCUSSION TITLE
Demo Package Errors with Authentication and 2FA

### DISCUSSION THREAD

**Discussion by elmarputz** [pimcore/pimcore#13219] [Q&A / Support]
After installing the latest Demo Package Version (10.5) on PHP 8.1, i tried to setup 2FA but it simply does not work. 

- Enabled the 2FA checkbox in the user profile 
- After reloading the backend no change in login form - 2FA does not trigger
- Fresh browser instance with cleared caches does not work either

I cleared all caches and temporary files, doesn't help. 

Setup: 
- Apache / PHP 8.1 FPM
- Redis Cache
- Maria DB 10.5

Thanks in advance!
Elmar

**Comment by dvesh3** [comment]
It seems like a bug with new authenticator. issue created https://github.com/pimcore/pimcore/issues/13220
